delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/wiring/AclsModule.scala

```diff
@@ -26,7 +27,10 @@ import monix.execution.Scheduler
 object AclsModule extends ModuleDef {
   implicit private val classLoader: ClassLoader = getClass.getClassLoader
 
-  make[EventLog[Envelope[AclEvent]]].fromEffect { databaseEventLog[AclEvent](_, _) }
+  make[EventLog[Envelope[AclEvent]]].fromEffect {
+    (flavour: DatabaseFlavour, as: ActorSystem[Nothing], _: DatabaseDefinitions) =>
```

Comment:
There should be a way to define a dependency to DatabaseDefinitions without repeating it.
I have looked in distage without finding something useful but what if when making the database flavour we would also run an effect that would create the tables if asked ?

Reply:
the way I've seen to enforce order is to add it to the dependencies as I did (in distage).
However I can try what you're suggesting.